doc/source/whatsnew/v0.16.2.txt

```diff
@@ -25,6 +25,8 @@ New features
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- Fix undesired UX behavior of DataFrame output in IPython Notebook. (:issue:`10231`)
```

Comment:
yeh, this needs a bit more explanation. Possibly even a screen shot. Its *seems* trivial, but people complain about UX changes the most!